DEVELOPMENT.md

```diff
@@ -118,7 +118,7 @@ By default, the Remix `rollup` build will strip any `console.debug` calls to avo
 REMIX_DEBUG=true yarn watch
 ```
 
-**`REMIX_LOCAL_DEV_OUTPUT_DIRECTORY`**
+**`REMIX_LOCAL_BUILD_DIRECTORY`**
```

Comment:
This will need to be tested still

Reply:
I gave it a quick run and it looks good to me - copying files to the right spots in my sibling app directory:

<img width="999" alt="Screen Shot 2022-06-20 at 9 01 55 PM" src="https://user-images.githubusercontent.com/1609022/174695054-66fca84b-45fe-4f3b-a3c5-33ed259c7ca9.png">


Saw one rollup complaint (in `yarn watch` only) about the empty `deno` chunk but I see that on dev as well so I don't think it's related.  It also doesn't break the `yarn watch` command.

<img width="875" alt="Screen Shot 2022-06-20 at 9 00 21 PM" src="https://user-images.githubusercontent.com/1609022/174694998-5df1a92f-48ce-4439-9bbb-4a43b3eceb9a.png">

Reply:
@brophdawg11 The "Generated an empty chunk" part is expected, but the "You must specify "output.file"..." part looks like it might be new since the Rollup upgrade. I think previously Rollup required an input, but not an output.


For Deno we just want to copy things with the `copy` plugin, so we can set `output.file` to some dummy location and that should work. A little hacky, but let's use use Rollup to build _all_ our packages instead of treating `remix-deno` specially.

Reply:
It's got something do to with `watch` mode - it happens both on `dev` and this branch but only when you run `yarn watch`, not `yarn build`.  Digging in a bit more now